docs/source/convert.rst

```diff
@@ -81,6 +81,14 @@ all files from the same deployment.
    The ``EchoData`` instance contains all the data unpacked from the raw file,
    so it is a good idea to clear it from memory once done with conversion.
 
+.. attention::
+    In version 0.6.2 of echopype we improved the in-memory usage of ``open_raw``
+    by allowing users to directly write variables that may consume a large amount of memory
+    into a temporary zarr store (see `#774 <https://github.com/OSOceanAcoustics/echopype/pull/774>`_).
+    This feature is accessible through ``open_raw`` via arguments ``offload_to_zarr`` and ``max_zarr_mb``
+    and is only available for the following echosounders: EK60, ES70, EK80, ES80, EA640.
+    See [API references](api) for usage.
```

Comment:
@leewujung is this the correct way to reference the api? This looks like markdown. I think we need to do something like :doc:\`api\`, right?

Reply:
Well, that's what the Readthedocs preview build is for :wink:. In this case, here's the preview: https://echopype--784.org.readthedocs.build/en/784/convert.html#conversion-operation
And it looks like the link is wrong. The page is in rst format, so the link should be rst type. I don't *think* that's :doc:`api`, but you can test it locally. Sorry, I don't have time right now to chime in with the correct format.

Reply:
I think api would just work but it does not seem to work in the admonition, so will have to test this. I have the build mechanism locally and can test this out and finish this PR.

Reply:
So a variation of what @b-reyes said :doc:`API references <api>` does work, and then I thought it's better to link to the exact function reference, so I ended up adding a label and link to that label.